[IMP] ecommerce: update product/attribute page (#12922)
odoo/documentation: content/applications/websites/ecommerce/products.rst

Customization
-------------

To customize a product page, click :guilabel:`Edit` in the upper right corner of the page. Use the

> :guilabel:`Blocks` tab to add

Reviewer: you can actually break a line inside any rst role (such as :doc:, :ref:, :guilabel:, etc.) at a space so :doc:`building blocks can go on line 158 and you can break the line after that.

Customization
-------------

To customize a product page, click :guilabel:`Edit` in the upper right corner of the page.

.. tip::
   You can edit any text on your website by clicking on it.

From here, you can add an additional, detailed product description especially for marketing purposes
that will only be visible on your ecommerce product page. To do so, click the text field beneath
the **sales description** located right under the product name. You can use the `/` command to add images,

> tables and other features.

Author: @auva-odoo ok! :)

- :guilabel:`Select Quantity`: Toggle the switch to allow customers to select the product quantity
  they want to purchase.

- :guilabel:`Tax Indication`: Toggle the switch to indicate if the price is
  :ref:`VAT included or excluded <ecommerce-price-management-tax-display>`.

- :guilabel:`Variants`: Show all possible product

>   :ref:`variants <ecommerce/products/product-variants>` vertically as a :guilabel:`Products List`

Reviewer: same as above :wink:

.. _ecommerce/products/image-customization:

Product images and videos
-------------------------

You can add additional media items to provide visual resources (i. e. images and videos) and information on
your product.

Navigate to the :ref:`product form <ecommerce/products/product-form>`, then go to the :guilabel:`Sales` tab
and click :guilabel:`Add a media` under the :guilabel:`Extra Product Media` section. In the resulting
:guilabel:`Create Extra Product Media` pop-up window, enter the :guilabel:`Image Name`, and add a video URL

> or click the :icon:`fa-pencil` :guilabel:`(pencil)` icon that appears when the cursor hovers
> over the :icon:`fa-camera` :guilabel:`(camera)` icon to upload images from your local drive.

Reviewer: ```suggestion
or hover your mouse over the :icon:`fa-camera` :guilabel:`(camera)` icon and click the :icon:`fa-pencil` :guilabel:`(pencil)` icon to upload images from your local drive.
```
(same as above: to try and list the actions in chronological order to make the steps easier to follow)

Author: Noted

The media resources will appear on the ecommerce product page where you can adapt their layout.

> To do so, go to the ecommerce product page, click :guilabel:`Edit` and select an image. In the :guilabel:`Customize`

Author: Thx
